src/status_im/commands/handlers/loading.cljs

```diff
@@ -95,18 +99,36 @@
 
 (defn add-commands
   [db [id _ {:keys [commands responses autorun]}]]
-  (let [account    @(subscribe [:get-current-account])
-        commands'  (filter-forbidden-names account id commands)
-        responses' (filter-forbidden-names account id responses)]
-    (-> db
-        (assoc-in [id :commands] (mark-as :command commands'))
-        (assoc-in [id :responses] (mark-as :response responses'))
-        (assoc-in [id :commands-loaded] true)
-        (assoc-in [id :autorun] autorun))))
+  (let [account        @(subscribe [:get-current-account])
+        commands'      (filter-forbidden-names account id commands)
+        global-command (:global commands')
+        commands''     (apply dissoc commands' [:init :global])
+        responses'     (filter-forbidden-names account id responses)]
+    (cond-> db
+
+            (get-in db [:chats id])
+            (update-in [:chats id] assoc
+                       :commands (mark-as :command commands'')
+                       :responses (mark-as :response responses')
+                       :commands-loaded true
+                       :autorun autorun
+                       :global-command global-command)
```

Comment:
That would simplify a lot of things, but @jarradh mentioned that we can use global command like `@bot command-name` to call some not global command. This actually will work, but i'm not sure that it looks cool.

On the other hand, if we allow an arbitrary number of global commands this can be a problem because of malicious bots that will be able  to fill suggestions list with useless commands...

We have another option: some configuration screen where a user can choose which bot's commands should be `/global`, like some permissions for commands, but i'm not sure thta we can include such thing in our `epic/chat-api` scope.